samples/meetings-transcription/csharp/MeetingTranscription/Models/Transcripts.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="Transcripts.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// </copyright>
+
+namespace MeetingTranscription.Models
+{
+    using System;
+
+    public class Transcripts
```

Comment:
Transcript

Reply:
Actually it is already present in BotBuilder namespace and showing ambiguous error

Reply:
Not sure if I understand the response completely, but Transcript**_s_** would be confusing. You might want to fix this, may be through fully qualified name?